src/BaGet/Startup.cs

```diff
@@ -23,32 +24,42 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.ConfigureBaGet(Configuration, httpServices: true);
+
+            // In production, the UI files will be served from this directory
+            services.AddSpaStaticFiles(configuration =>
+            {
+                configuration.RootPath = Path.Combine("Baget.Ui", "dist");
+            });
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IHostingEnvironment env)
         {
-            if (env.IsDevelopment())
+            var scopeFactory = app.ApplicationServices
+                .GetRequiredService<IServiceScopeFactory>();
+
+            using (var scope = scopeFactory.CreateScope())
             {
-                app.UseDeveloperExceptionPage();
-                app.UseStatusCodePages();
+                IContext context = scope.ServiceProvider
+                    .GetRequiredService<IContext>();
 
-                // Run migrations automatically in development mode.
-                var scopeFactory = app.ApplicationServices
-                    .GetRequiredService<IServiceScopeFactory>();
+                // Always ensure the database is created.
+                context.Database.EnsureCreated();

```

Comment:
Why did you add a call to `EnsureCreated`? From [the documentation](https://docs.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.infrastructure.databasefacade.ensurecreated?view=efcore-2.1), it appears that `EnsureCreated` doesn't play well with migrations.